### ISSUE TITLE
bge-reranker微调后加载脚本会改变吗

### ISSUE THREAD

**Issue by zhaobinNF** [FlagOpen/FlagEmbedding#144]
bge-reranker微调后加载脚本还是下面这个嘛
import torch
from transformers import AutoModelForSequenceClassification, AutoTokenizer, BatchEncoding, PreTrainedTokenizerFast

tokenizer = AutoTokenizer.from_pretrained('BAAI/bge-reranker-base')
model = AutoModelForSequenceClassification.from_pretrained('BAAI/bge-reranker-base')
model.eval()

pairs = [['what is panda?', 'hi'], ['what is panda?', 'The giant panda (Ailuropoda melanoleuca), sometimes called a panda bear or simply panda, is a bear species endemic to China.']]
with torch.no_grad():
    inputs = tokenizer(pairs, padding=True, truncation=True, return_tensors='pt', max_length=512)
    scores = model(**inputs, return_dict=True).logits.view(-1, ).float()
    print(scores)

**Comment by staoxiao** [comment]
是的，把`BAAI/bge-reranker-base`换成微调后的模型存储路径即可。
